target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Fri Jun 26 15:42:51 UTC 2020
+#Mon Jun 29 16:38:32 UTC 2020
 version=1
 groupId=com.google.sps
 artifactId=sdktiktok
```

Comment:
I think we didn't decide on this name. Maybe change to firebasesdkdash or something.

Reply:
We did not, I could change when we decide upon the name.